index.bs

```diff
@@ -2307,7 +2283,6 @@ To determine if <dfn>poses may be reported</dfn> to an {{XRSession}} |session|,
 
   1. Let |document| be the document that owns |session|.
   1. If the request does not originate from |document|, return <code>false</code>.
-  1. If |document| is not [=active and focused=], return <code>false</code>.
```

Comment:
Hmm, isn't removing this a behavior change?

Reply:
Actually, we handle this in the "currently focused area" section.

Reply:
Yes, this is a behavior change, in that in the old spec there was a race condition, if a user gave consent (e.g. by clicking on en "enter VR" button) then quickly changed the focus to an iframe, the old check would fail but the new check succeeds.

Reply:
I don't understand: this is a per-frame thing about when poses can be reported. What per-frame behavior is here now?

Reply:
Shouldn't tabbing out of a tethered session pause the session?

Reply:
It is weird that tabbing out would cause an exception to be thrown, though. In this case it might be better to report a null pose (something I plan to bring up later)

Reply:
I still think we shouldn't base pose data on the state of the originating document, since the user can't see the originating document while they've got a headset on

I added back text requiring document visibility, we should discuss this in a separate issue.